index.html

```diff
@@ -0,0 +1,276 @@
+
+    
+    <link rel="stylesheet" href="style.css">
+
+    <title>Sanam's Portfolio</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    
+<section class="head">
+      <header class="header">
+        <nav class="nav-menu">
+            <a href="#"> <img src="images\Logo Placeholder.svg" alt="logo"></a>
+
+            <div class="nav-menu">
+                <ul class="menu-desktop" href="#">
+                    <li><a href="my-work">Portfolio</a></li>
+                    <li><a href="about-me">About</a></li>
+                    <li><a href="my-work">Contact</a></li>
+                </ul>
+                <img src="images\Union.svg" alt="menu-icon" class="menu hamburger-menu"/>
+            </div>
+
+            <div class="menu-hide mobile-menu-container">
+                <div class="mobile-menu">
+                  <div class="close-menu">
+                    <img class="close-menu-icon" src="images\Icon.svg" alt="close-icon">
+                  </div>
+                  <ul class="mobile-menu-item-list">
+                    <li class="mobile-menu-item"><a href="#my-works">Portfolio</a></li>
+                    <li class="mobile-menu-item"><a href="#about-me">About</a></li>
+                    <li class="mobile-menu-item"><a href="#contact-me">Contact</a></li>
+                  </ul>
+                  </div>
+                  </div>
+        </nav>
+      </header>
+
+      <section class="headline-section blur-bg">
+        <div class="h-container h-content">
+          <div class="h-main">
+            <h1>I'm Sanam,<br />Glad to see you</h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through <br> some of my work and experience! If you like what you see and have a project you need <br> coded, don't hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+
+            <ul class="a-socials-links">
+                <li>
+                    <a href="https://twitter.com"><img src="images\Vector.svg" alt="twitter-icon"></a>
+                </li>
+
+                <li>
+                    <a href="https://www.linkedin.com"><img src="images\Linkedin icon.svg" alt="linkedin-icon"></a>
+                </li>
+
+                <li>
+                    <a href="#"><img src="images\Vector (1).svg" alt="intelli-icon"></a>
+                </li>
+
+                <li>
+                    <a href="https://github.com"><img src="images\Vector (2).svg" alt="github-icon"></a>
+                </li>
+
+                <li>
+                    <a href="#"><img src="images\Vector (3).svg" alt="medium-icon" /></a>
+                  </li>
+
+            </ul>
+          </div>
+        </div>
+
+
+</section>

```

Comment:
Kindly follow the Figma template in your design. Presently the background image is not like what we have in the design 😞 . Also, kindly make use of the font size and the colors as stated in the design 🙏 .
| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/227647269-cb4d5684-1d26-4b7c-b3a3-affdf861c680.png">|<img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/227647327-8728e63a-eeef-42e0-816a-e0c285aec74d.png">|